docs/plugin_developer_guide.md

```diff
@@ -1,96 +1,96 @@
-# Bluemix CLI Plugin Developer's Guide
+# Bluemix CLI Plug-in Developer's Guide
 
-This guide introduces how to develop a Bluemix CLI plugin by using utilities and libraries provided by Bluemix CLI SDK. It also covers specifications including wording, format and color of the terminal output that we highly recommend developers to follow.
+This guide introduces how to develop a Bluemix CLI plug-in by using utilities and libraries provided by Bluemix CLI SDK. It also covers specifications including wording, format and color of the terminal output that we highly recommend developers to follow.
 
-## 1. Plugin Context Management
+## 1. Plug-in Context Management
 
-Bluemix CLI SDK provides a set of APIs to register and manage plugins. It is similar to  CF plugin interface, but has more Bluemix specific extensions.
+Bluemix CLI SDK provides a set of APIs to register and manage plug-ins. It is similar to CF plug-in interface, but has more Bluemix specific extensions.
 
-### 1.1. Register a New Plugin
+### 1.1. Register a New Plug-in
 
-_Step 1:_ Define a new struct for Bluemix CLI plugin and associate Run and GetMetadata methods to that struct:
+1.  Define a new struct for Bluemix CLI plug-in and associate `Run` and `GetMetadata` methods to that struct:
```

Comment:
Also, throughout the doc I put command snippets like `Run` and `GetMetadata` here in ticks so that they'd render different from paragraph text for improved scanning.